test/e2e/data/kustomize/kcp-remediation/md.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: MachineDeployment
+metadata:
+  name: "${CLUSTER_NAME}-md-0"
+spec:
+  template:
+    metadata:
+      labels:
+        "e2e.remediation.label": ""
```

Comment:
I think this is only required for md-remediation. Also kustomization.yaml doesn't include it.

Reply:
Thanks, well spotted!